### ISSUE TITLE
No support for URL in plugin

### ISSUE THREAD

**Issue by wonkyto** [elastic/puppet-logstash#337]
Currently there is no url support in logstash::plugin like there is in elasticsearch::plugin module: https://github.com/elastic/puppet-elasticsearch/blob/master/manifests/plugin.pp

This is an issue if you would like to install the x-pack plugin from a remote URL, even though it is possible to do this via the command line eg:
```
/usr/share/logstash/bin/logstash-plugin install https://artifacts.elastic.co/downloads/packs/x-pack/x-pack-5.3.0.zip
```
In our case we actually have the x-pack file stored on an internal webserver and don't rely on the upstream elastic.co location but the point is the same.

**Comment by jarpy** [comment]
Thanks. I've patched in this support now, and released it on Puppet Forge as part of module version 5.1.0.

**Comment by kimor79** [comment]
I don't think this was merged into master. I only see the `file:/` changes in plugin.pp

**Comment by jarpy** [comment]
Eek! You're right. Sorry everyone.

**Comment by jarpy** [comment]
This is now merged and released in 5.2.0.

Thanks for the tipoff, @kimor79.
